rollup.config.js

```diff
@@ -3,6 +3,7 @@ import resolve from '@rollup/plugin-node-resolve';
 import typescript from '@rollup/plugin-typescript';
 import dts from 'rollup-plugin-dts';
 import peerDepsExternal from 'rollup-plugin-peer-deps-external';
+import scss from 'rollup-plugin-scss';
```

Comment:
This import isn't referenced anywhere, is it a side-effect-only import?

Reply:
It is referenced in plugins

Reply:
Oh I see it now, sorry I missed that!